pkg/migrations/op_add_column_test.go

```diff
@@ -255,6 +273,58 @@ func TestAddColumn(t *testing.T) {
 				}, res)
 			},
 		},
+		{
+			name: "add column to non-empty table without primary key",
+			migrations: []migrations.Migration{
+				{
+					Name: "01_add_table",
+					Operations: migrations.Operations{
+						&migrations.OpCreateTable{
+							Name: "users",
+							Columns: []migrations.Column{
+								{
+									Name: "name",
+									Type: "varchar(255)",
+								},
+							},
+						},
+					},
+				},
+				{
+					Name: "02_add_column",
+					Operations: migrations.Operations{
+						&migrations.OpAddColumn{
+							Table: "users",
+							Column: migrations.Column{
+								Name:     "age",
+								Type:     "integer",
+								Nullable: true,
+							},
+							Up: "1",
+						},
+					},
+				},
+			},
+			afterRollback: func(t *testing.T, db *sql.DB, schema string) {
```

Comment:
I think the test should have an `afterStart` part too - even if it's very simple.